src/deploy/index.js

```diff
@@ -82,8 +83,8 @@ module.exports = async ({
     return { status: 'Succeeded' }
   }
 
-  if (!specificFilesMode && destructive) {
```

Comment:
here there is something that it was unexpected: ! specificFilesMode should be specificFilesMode?